packages/kbn-shared-ux-components/src/toolbar/buttons/add_from_library/add_from_library.stories.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { ToolbarButton } from '../primary/primary';
+import mdx from './add_from_library.mdx';
+
+export default {
+  title: 'Toolbar/Buttons/Add From Library Button',
+  description: 'An implementation of the solution toolbar primary button',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Component = () => {
+  return <ToolbarButton label="Add from library" iconType="folderOpen" />;
```

Comment:
Should these props be controls in storybook? Is the story for the actual `AddFromLibrary` missing?

Reply:
Following now thanks! [7140323](https://github.com/elastic/kibana/pull/127605/commits/7140323d594fe4b7dbea10e5ea9827ac6e3326ed) for changes